.github/workflows/optimize-release-optimize-c8-only.yml

```diff
@@ -255,12 +271,14 @@ jobs:
           echo "Tagging optimize release docker image with version ${RELEASE_VERSION}"
           tags=("${DOCKER_IMAGE_TEAM}:${RELEASE_VERSION}")
           tags+=("${DOCKER_IMAGE_DOCKER_HUB}:${RELEASE_VERSION}")
+          tags+=("${DOCKER_INTERNAL_IMAGE_DOCKER_HUB}:${RELEASE_VERSION}")
 
           # Major and minor versions are always tagged as the latest
           if [ "${MAJOR_OR_MINOR}" = true ] || [ "${DOCKER_LATEST}" = true ]; then
               echo "Tagging optimize release docker image with \`${DOCKER_LATEST_TAG}\`"
               tags+=("${DOCKER_IMAGE_TEAM}:${DOCKER_LATEST_TAG}")
               tags+=("${DOCKER_IMAGE_DOCKER_HUB}:${DOCKER_LATEST_TAG}")
+              tags+=("${DOCKER_INTERNAL_IMAGE_DOCKER_HUB}:${DOCKER_LATEST_TAG}")
```

Comment:
No need to push latest to the internal repo, CVE-Registry is configured to ignore latest

Reply:
Okay, I deleted it